### PR TITLE
doc: updated studio user guide

### PR DESCRIPTION
**Pull request by oldLady344**
- added (translated) docs about limitations, what-is, deployment, and connection.  
- updated links. 
- to keep consistent, replaced "the XX tab page" with "the XX page". 
- to keep consistent, replaced "the XX icon" with "the button (icon)".

### CODE REVIEW

File: nebula-studio/about-studio/st-ug-what-is-graph-studio.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# What is Nebula Graph Studio
+
+Studio (short for Nebula Graph Studio) is a browser-based visualization tool to manage Nebula Graph databases. It provides graphical user interface for you manipulate graph schemas, import data, explore graph data, and run nGQL statements to query data. With Studio, you can quickly become graph exploration expert from scratch.
```
Comment (Amber1990Zhang) [inline nebula-studio/about-studio/st-ug-what-is-graph-studio.md:3]: to manage Nebula Graph

Comment (Amber1990Zhang) [inline nebula-studio/about-studio/st-ug-what-is-graph-studio.md:3]: for you to manipulate graph schemas

File: nebula-studio/about-studio/st-ug-what-is-graph-studio.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Release distributions
+
+Studio has two release distributions:
```
Comment (Amber1990Zhang) [inline nebula-studio/about-studio/st-ug-what-is-graph-studio.md:7]: Studio has two releases?

File: nebula-studio/install-configure/st-ug-install.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+1. Download the Studio deployment source code.
+
+    ```bash
+    git clone https://github.com/vesoft-inc/nebula-web-docker.git
+    ```
+
+2. Change to the `nebula-web-docker` directory.
+
+    ```bash
+    cd path/to/nebula-web-docker
```
Comment (Amber1990Zhang) [inline nebula-studio/install-configure/st-ug-install.md:29]: cd nebula-web-docker

File: nebula-studio/install-configure/st-ug-connect.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## Next to do
+
+When Studio is connected to Nebula Graph successfully, you can do these operations:
+
+- If your account has GOD or ADMIN privilege, you can create a schema on the **[Console](../quick-start/st-ug-create-schema.md)** page or on the **[Schema](../manage-schema/st-ug-crud-space.md)** page.
+
+- If your account has GOD, ADMIN, DBA, or USER privilege, you can bulk import data on the **[Import](../quick-start/st-ug-import-data.md)** page or insert data one by one on the **Console** page.
+
+- If your account has GOD, ADMIN, DBA, USER, or GUEST privilege, you can operate queries on the **Console** page or analyze data on the **Explore** page.
```
Comment (Amber1990Zhang) [inline nebula-studio/install-configure/st-ug-connect.md:47]: add built in role doc here

File: nebula-studio/about-studio/st-ug-what-is-graph-studio.md (hunk 2 above)
Comment (oldLady344) [reply nebula-studio/about-studio/st-ug-what-is-graph-studio.md:7]: > Studio has two releases?

Yes. Docker-based Studio and Studio on Cloud.

File: nebula-studio/install-configure/st-ug-connect.md (hunk 4 above)
Comment (oldLady344) [reply nebula-studio/install-configure/st-ug-connect.md:47]: > add built in role doc here

I will change "operate queries" to "retrieve data". All these roles have the "read data" privilege.

File: nebula-studio/manage-schema/st-ug-crud-edge-type.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Operate edge types
 
-After a graph space is created in Nebula Graph, you can create edge types. With Studio, you can choose to use the **Console** tab page or the **Schema** tab page to create, retrieve, update, or delete edge types. This article only introduces how to use the **Schema** tab page to operate edge types in a graph space.
+After a graph space is created in Nebula Graph, you can create edge types. With Studio, you can choose to use the **Console** page or the **Schema** page to create, retrieve, update, or delete edge types. This article only introduces how to use the **Schema** page to operate edge types in a graph space.
 
 ## Studio version
```
Comment (whitewum) [inline nebula-studio/manage-schema/st-ug-crud-edge-type.md:6]: line 10
`-d`

**Review by whitewum — APPROVED** [review]
`docker-compose pull && docker-compose up -d`

**Comment by whitewum** [comment]
please check for `up -d`

**Review by randomJoe211 — APPROVED** [review]
Great!
See my comments for suggestions of optimization.

File: nebula-studio/about-studio/st-ug-limitations.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+- `CHANGE PASSWORD`
+- `DROP USER`
+- `GRANT ROLE`
+- `REVOKE ROLE`
+
+For more information about the preceding statements, see [Nebula Graph Database Manual](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/4.account-management-statements/alter-user-syntax/).
+
+## Browser
+
+We recommend that you use Chrome to get access to Studio.
```
[the remark this answers is not in the document]
Comment (oldLady344) [reply nebula-studio/about-studio/st-ug-limitations.md:48]: > get access to => access

To write for a global audience, it's better not to use "access" as a verb.

**Pull request merged**